docs/src/modules/components/ApiPage.js

```diff
@@ -39,6 +44,60 @@ const Table = styled('table')(({ theme }) => {
     // as a workaround, use negative margin with overflow `hidden` on the parent
     marginLeft: -1,
     marginRight: -1,
+    '.anchor-link-style': { display: 'none' },
+    tr: {
+      scrollMarginTop: 'calc(var(--MuiDocs-header-height) + 32px)',
```

Comment:
This might no longer be needed with this fix: https://github.com/mui/material-ui/pull/36972#discussion_r1174580420